zipkin-collector/scribe/pom.xml

```diff
@@ -58,5 +58,12 @@
       <version>${awaitility.version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
```

Comment:
should we do this in the parent pom, or is it because we have something that isn't jupiter...

Reply:
> should we do this in the parent pom, or is it because we have something that isn't jupiter...

right, there are few (benchmarks, zipkin-lens) that do not use JUnit